bitrise.schema.json

```diff
@@ -370,6 +373,15 @@
       "additionalProperties": false,
       "type": "object"
     },
+    "WorkflowStageConfig": {
```

Comment:
We only have `*Model` object in this file and this is called `WorkflowStageConfig`. To me this looks like an alternative model for the workflow so what if we call it `WorkflowStageModel` instead?